src/display/annotation_storage.js

```diff
@@ -19,6 +19,7 @@
 class AnnotationStorage {
   constructor() {
     this._storage = new Map();
+    this.modified = false;
```

Comment:
Make this private, i.e., `this._modified`, and add a getter `get modified` for this property similar to the `size` property for external usage.